### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by VishakaSekar**

### CODE REVIEW

**Comment by VishakaSekar** [comment]
I've made the changes mentioned in CR1. Requesting review of CR1 changes.

File: src/main/java/com/appdynamics/extensions/aws/redshift/RedshiftMetricsProcessor.java [hunk 1]
```diff
@@ -140,32 +140,44 @@ private void arrangeMetricPathHierarchy(StringBuilder partialMetricPath, Map<Str
...
         //<Account> | <Region> | Cluster Identifier | <ClusterId> | Node ID | <NodeId> |
         if(dimensionValueMap.get(nodeIDDimension)!= null){
             buildMetricPath(partialMetricPath, true, nodeIDDisplayName,
                     dimensionValueMap.get(nodeIDDimension));
         }
 
-        // #TODO In the comment blocks below, you have only added the case when Node Id is present, can you please add the other case/cases also?
-        //<Account> | <Region> | Cluster Identifier | <ClusterId> | Node ID | <NodeId> | Latency | <LatencyValue> |
+        // if this is a cluster level metric,
+        // <Account> | <Region> | Cluster Identifier | <ClusterId> |
```
Comment (venkatakonala) [inline src/main/java/com/appdynamics/extensions/aws/redshift/RedshiftMetricsProcessor.java]: Looks like there can be latency under Cluster. Please check and make the appropriate changes.

Comment (VishakaSekar) [reply src/main/java/com/appdynamics/extensions/aws/redshift/RedshiftMetricsProcessor.java]: Yes, will make that change to the comment block

**Comment by VishakaSekar** [comment]
Made the changes to comment block for latency metric. Requesting review

**Pull request merged**